plugin/route53/setup.go

```diff
@@ -91,12 +86,12 @@ func setup(c *caddy.Controller) error {
 				c.RemainingArgs() // eats args
 			case "credentials":
 				if c.NextArg() {
-					sharedProvider.Profile = c.Val()
+					os.Setenv("AWS_PROFILE", c.Val())
```

Comment:
This will potentially mess with other `route53` plugins using different credentials no? May need to use https://pkg.go.dev/github.com/aws/aws-sdk-go/aws/session#Options with `session.NewSessionWithOptions` call.